src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2694,18 +2694,13 @@ private ConstructorDeclarationSyntax ParseConstructorDeclaration(string typeName
             {
                 var paramList = this.ParseParenthesizedParameterList(allowThisKeyword: false, allowDefaults: true, allowAttributes: true);
 
-                ConstructorInitializerSyntax initializer = null;
-                if (this.CurrentToken.Kind == SyntaxKind.ColonToken)
-                {
-                    bool isStatic = modifiers != null && modifiers.Any((int)SyntaxKind.StaticKeyword);
-                    initializer = this.ParseConstructorInitializer(name.ValueText, isStatic);
-                }
+                var initializer = this.CurrentToken.Kind == SyntaxKind.ColonToken
+                    ? this.ParseConstructorInitializer()
+                    : null;
 
-                BlockSyntax body;
-                ArrowExpressionClauseSyntax expressionBody;
-                SyntaxToken semicolon;
-                this.ParseBlockAndExpressionBodiesWithSemicolon(out body, out expressionBody, out semicolon,
-                                                                requestedExpressionBodyFeature: MessageID.IDS_FeatureExpressionBodiedDeOrConstructor);
+                this.ParseBlockAndExpressionBodiesWithSemicolon(
+                    out var body, out var expressionBody, out var semicolon,
```

Comment:
>var [](start = 24, length = 3)

According to our coding guidelines, ```var``` cannot be used in this manner, the type isn't explicitly stated in the initializer. Also, this is a style change for the sake of the style change. Please revert.